addons/account/models/chart_template.py

```diff
@@ -335,6 +335,9 @@ def tax_template_changed(tax, template):
                             }])
                             account = existing_account
 
+                    # avoid to override user custom settings & partial reconcile error
+                    if 'reconcile' in values:
+                        del values['reconcile']
```

Comment:
```suggestion
                    # Prevents overriding user setting & raising a partial reconcile error.
                    values.pop('reconcile', None)
```
Slightly more concise :nerd_face:

Reply:
done